services/web/server/src/simcore_service_webserver/projects/project_models.py

```diff
@@ -1,10 +1,12 @@
 from enum import Enum
-from typing import Any, Dict, Optional
+from typing import Any, Dict, Optional, Tuple
 
+from aiopg.sa.result import ResultProxy
 from simcore_postgres_database.models.projects import ProjectType
 
 # TODO: extend
 ProjectDict = Dict[str, Any]
+ProjectProxy = ResultProxy
```

Comment:
this one, I'm not too sure whether it's good. I mean a ResultProxy is something coming from aiopg and probably we should have converted to a dict instead of passing it around (at least that is my impression). My point is that I'm not sure passing ProjectProxy is very clear.
Whatever... it's just a thought.

Reply:
@sanderegg actually it is a mistake. It is a ``RowProxy`` and not ``ResultProxy``

I need to do all these aliases because right now it is very difficult to figure out what "type" of datastructure is holding the ``Project``. 

NOTE: aiopg does not covert it to dict ... see some trials i did in [test_utils_aiopg_orm](https://github.com/ITISFoundation/osparc-simcore/blob/master/packages/postgres-database/tests/test_utils_aiopg_orm.py#L278)